app/views/user/layouts/sessions.html.erb

```diff
@@ -2,4 +2,5 @@
   <p></p>
 <% end %>
 
+<%= render template: "user/layouts/application" %>
```

Comment:
Is this because the `user/layouts/application` layout includes stuff we don't want for a not-logged-in user, so with with the `javascript_include_tag 'user/application'` we include the necessary javascripts but not the whole layout?